### DISCUSSION TITLE
max_text_length 这个默认的25 也可以识别单行文字超过25个字符的图片，那这个参数为什么要设置？

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#12593] [Q&A]
#### 运行环境 / Runtime Environment
- OS:
- Paddle: 2.3
- PaddleOCR: 2.7.5

#### 附件 / Appendix

max_text_length 这个默认的25 也可以识别单行文字超过25个字符的图片，那这个参数为什么要设置？
测试图片如下，仅识别，可以正确识别25个字符以上的字符
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/41010669/b6c1b619-2d1a-4932-be92-06c7bb8c044e)

**Comment by GreatV** [comment]
- https://github.com/PaddlePaddle/PaddleOCR/issues/12065#issuecomment-2101865288
